.github/workflows/checks-and-tests.yml

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+# at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+# LICENSE and NOTICE for details. LLNL-CODE-806117.
+#
+# This file is part of the MFEM library. For more information and source code
+# availability visit https://mfem.org.
+#
+# MFEM is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+name: checks-and-tests
```

Comment:
Maybe a comment describing how `checks-and-tests` is different than `repo-checks` would be helpful.

Reply:
Good point.